src/library/returnBook/ReturnBookUI.java

```diff
@@ -1,101 +1,100 @@
 package library.returnBook;
+
 import java.util.Scanner;
 
 
 public class ReturnBookUI {
 
-    private enum uI_sTaTe { INITIALISED, READY, INSPECTING, COMPLETED };
+    private enum UiState {INITIALISED, READY, INSPECTING, COMPLETED};
+
+    private ReturnItemControl control;
+
+    private Scanner input;
 
-    private ReturnItemControl CoNtRoL;
-    private Scanner iNpUt;
-    private uI_sTaTe StATe;
+    private UiState state;
```

Comment:
#PR42 - D4: Please rename/refactor ReturnItemUIState variable 'state' to 'uiState' as per class diagram

Reply:
[110d34d](https://github.com/elliotmitchum/ITC205/pull/42/commits/110d34d17174af49be196ef13a74b9516fef2a17)